lib/heartcheck/controllers/inspect.rb

```diff
@@ -0,0 +1,20 @@
+module Heartcheck
+  module Controllers
+    class Inspect < Base
+      def index
+        results = {
+          application_name: Rails.application.class.name,
+          environment: ENV['RAILS_ENV'],
+          dependencies: []
+        }
+
+        checks = Heartcheck.checks
+        results[:dependencies] += checks.reduce([]) do |acc, elem|
+          acc << elem.uri_info
```

Comment:
Today is possible to have multiple checks in the same dependency, for example, to check the response of `/health_check` in http://apix.com and to check other thing in the same api.

In this case, this `reduce` would return the same `uri_info` twice. How would you deal with this?

Reply:
I'm not really sure. Maybe we can test and solve this duplication before returning this datastructure, or leave it to the endpoint's consumer to solve it. What do you think is the best solution, here?

Reply:
I've added a .uniq call in the loop's end. It dives into the hash an compare atomic values, so it seems fine.